tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37
+    py27,py34,py35,py36,py37,py38
```

Comment:
Time to drop py34, I'd say.

Reply:
Yes. Done.
I will force push my branch.